src/main/java/pl/wedding/management/service/weddinghall/model/WeddingHall.java

```diff
@@ -0,0 +1,18 @@
+package pl.wedding.management.service.weddinghall.model;
+
+import lombok.*;
+
+import java.util.EnumSet;
+import java.util.Set;
+
+@Getter
```

Comment:
@Value already provides @Getter, and @AllArgsConstructor. As documentation says:
```
Equivalent to {@code @Getter @FieldDefaults(makeFinal=true, level=AccessLevel.PRIVATE) @AllArgsConstructor @ToString @EqualsAndHashCode}.
```